Documentation/ApiOverview/Context/_MyControllerUsingDependencyInjection.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyVendor\MyExtension\Controller;
+
+use TYPO3\CMS\Core\Context\Context;
+
+final class MyController
+{
+    /**
+     * @var Context
+     */
+    protected Context $context;
+
+    public function __construct() {}
```

Comment:
What is the purpose of an empty constructor?

Reply:
This is just an example to show that there can be an empty constructor.

Reply:
It has the purpose for the understanding to the user of this example to show that here are no constructor parameters needed.